### ISSUE TITLE
[ FEATURE ] Ticketsystem for adding notes to Shows

### ISSUE THREAD

**Issue by wimpyrbx** [pymedusa/Medusa#576]
A way to track comments or "todo"s for shows/seasons.
A lot of users are using SR/Medusa to not only download, but manage their shows collection.

I'll use myself as an example as I have quite a good number of shows and are currently in progress of replacing shows/seasons with other qualities.

it would be nice to be able to click "add ticket" on a show and write a comment like "season 4 is a mixture of 720p and SD. will replace when this weekend with full 720p season"

then this would show as an open ticket on said show.
once resolved the ticket can be closed.

open tickets would need to be able to be displayed on the DisplayShow page for each show, but also on a new page called "Show tickets" or something that would list all open tickets for easy access.

Basically it's a ticket system to keep track of stuff one would like to fix or is going to fix with collections in sickrage/medusa.

Thoughts?

**Comment by p0psicles** [comment]
I like the idea. But I personally wouldn't have much need for it. Maybe others do. I'll register it as a feature request. Although I don't expect it to have high prio within the team, as it would require changes to the db model, and quite some time designing/implementing.

**Comment by wimpyrbx** [comment]
thanks.

it would require a db change yes, however it would not meddle with current tables i would think? a new table with link to showid would suffice?

**Comment by p0psicles** [comment]
Yeah that would suffice

**Comment by labrys** [comment]
@wimpyrbx I was actually thinking this same thing on an episode basis.  e.g. S01E04 is hardsub, would rather an unsubbed.  Or for anime: S01E04 is a dub.

**Comment by wimpyrbx** [comment]
exactly. could like you said also be done on episode level as you say.
perhaps show, season and episode level all could work alongside eachother.

have both
* showid
* season
* episode

as fields to identify if it's a ticket for the entire show. a specific season or a specific season+episode.

this would allow for a hierarchy in the output when listing all open tickets as well i believe.

**Comment by labrys** [comment]
Added to master feature request list - discussion for feature will continue here even though issue is closed.
